x/ibc/07-tendermint/types/codec.go

```diff
@@ -24,3 +18,30 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(&MsgUpdateClient{}, "ibc/client/tendermint/MsgUpdateClient", nil)
 	cdc.RegisterConcrete(&MsgSubmitClientMisbehaviour{}, "ibc/client/tendermint/MsgSubmitClientMisbehaviour", nil)
 }
+
+// RegisterInterfaces registers the tendermint concrete evidence and client-related
+// implementations and interfaces.
+func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
```

Comment:
Does this function get called anywhere yet?

Reply:
I think I forgot to add the call to `x/ibc/types/codec.go` which calls it from the module manager callback. Nevertheless this just needs to be added in the followup pr anyways